Pong/pong/pong.h

```diff
@@ -0,0 +1,33 @@
+//
+//  pong.h
+//  Pong
+//
+//  Created by Kaji Tetsushi on 2/20/22.
+//  Copyright © 2022 Kaji Tetsushi. All rights reserved.
+//
+
+#ifndef pong_h
+#define pong_h
+
+#include <SDL2/SDL.h>
+#include <stdbool.h>
+#include <stdio.h>
```

Comment:
[L] Another touted advantage of using only one consolidated header file is that the dependencies on other libraries can be declared on the header file itself without cherrypicking them on every implementation detail.